benchmarks/game_of_life.correct

```diff
@@ -1,41 +1,41 @@
-5555555
-5001115
-5000005
-5100005
-5111115
-5101005
-5555555
+-------
```

Comment:
How much extra effort is it to use `+` for the corners? :P

Reply:
Too much : P

Reply:
wait not it's not!